tests/conftest.py

```diff
@@ -1,43 +1,17 @@
-import os
 
-import pytest
-from sqlalchemy.exc import ProgrammingError
-
-from ..log import LogConfiguration
-from ..model import (SessionManager,
-    Base,
-)
-
-
-@pytest.fixture(autouse=True, scope="session")
-def session_fixture():
-    # This will make sure we always connect to the test database.
-    os.environ['TESTING'] = 'true'
-
-    # Ensure that the log configuration starts in a known state.
-    LogConfiguration.initialize(None, testing=True)
-
-    # Drop any existing schema. It will be recreated when
-    # SessionManager.initialize() runs.
-    #
-    # Base.metadata.drop_all(connection) doesn't work here, so we
```

Comment:
Any idea why this didn't work beofre? It does work for me now in my testing, and it seems like an easier strategy, so I changed this to use `Base.metadata.drop_all()`. I can roll this back if that is undesirable.

Reply:
I think it didn't drop the materialized views because they weren't counted as 'tables'. Now we no longer use materialized views so it should be fine.